src/redux/rockets.js

```diff
@@ -1,15 +1,45 @@
 /* eslint-disable prefer-destructuring */
 const GETAPI = 'GET_API_DATA';
+const ADDRESERVATION = 'ADD_ROCKET_RESERV';
+const REMOVERESERVATION = 'REMOVE_ROCKET_RESERV';
```

Comment:
As per [ducks ](https://github.com/erikras/ducks-modular-redux)pattern rules it is recommended to use `npm-module-or-app/reducer/ACTION_TYPE` for naming action types.